mitosheet/dev/create_new_step.py

```diff
@@ -0,0 +1,379 @@
+""""
+The file contains utilities useful for generating a new
+step code. 
+
+This is useful as there is lots of boilerplate that needs
+to be written, and if we can cut it down to zero, that is
+ideal!
+"""
+
+import os
+from pathlib import Path
+from typing import Dict, Tuple
+
+# Constants useful in f strings, since they can't contain brackets
+OPEN_BRACKET = "{"
+CLOSE_BRACKET = "}"
+
+# Markers in the code that this script uses to find where to insert code
+IMPORT_MARKER = "# AUTOGENERATED LINE: IMPORT (DO NOT DELETE)"
+EXPORT_MARKER = "# AUTOGENERATED LINE: EXPORT (DO NOT DELETE)"
+TEST_MARKER = "# AUTOGENERATED LINE: TEST (DO NOT DELETE)"
+
+# Types that step parameters can use in a valid way
+VALID_PARAMETER_TYPES = [
+    'int', 'float', 'str', 'bool',
+    'ColumnID', 'List[ColumnID]', 
+    'Any'
+]
+
+def get_step_performers_folder() -> Path:
+    return Path('./mitosheet/step_performers')
+
+def get_code_chunk_folder() -> Path:
+    return Path('./mitosheet/code_chunks')
+
+def get_test_folder() -> Path:
+    return Path('./mitosheet/tests/step_performers')
+
+def get_step_name(original_step_name: str) -> str:
+    return original_step_name.lower().replace(' ', '_')
+
+def get_step_performer_name_and_import_statement(original_step_name: str) -> Tuple[str, str]:
+    step_name = get_step_name(original_step_name)
+    step_performer_name = original_step_name.replace(' ', '') + "StepPerformer"
+    return (
+        step_performer_name,
+        f'from mitosheet.step_performers.{step_name} import {step_performer_name}'
+    )
+
+def get_code_chunk_name_and_import_statement(original_step_name: str) -> Tuple[str, str]:
+    step_name = get_step_name(original_step_name)
+    code_chunk_name = original_step_name.replace(' ', '') + "CodeChunk"
+    return (
+        code_chunk_name,
+        f'from mitosheet.code_chunks.{step_name} import {code_chunk_name}'
```

Comment:
I think this should be {step_name}_code_chunk


<img width="780" alt="Screen Shot 2022-06-03 at 11 33 47 AM" src="https://user-images.githubusercontent.com/18709905/171889065-3b2db162-a38d-4b56-b21d-ca7ddcb6588e.png">

<img width="536" alt="Screen Shot 2022-06-03 at 11 34 14 AM" src="https://user-images.githubusercontent.com/18709905/171889525-8eb54d6c-200e-4e1c-a269-315914c0ab0d.png">